kernels/volk/volk_32f_stddev_and_mean_32f_x2.h

```diff
@@ -43,16 +43,19 @@
  *
  * \b Example
  * Generate random numbers with c++11's normal distribution and estimate the mean and
- * standard deviation \code int N = 1000; unsigned int alignment = volk_get_alignment();
+ * standard deviation
+ * \code
+ *   int N = 1000;
+ *   unsigned int alignment = volk_get_alignment();
  *   float* rand_numbers = (float*) volk_malloc(sizeof(float)*N, alignment);
  *   float* mean = (float*) volk_malloc(sizeof(float), alignment);
  *   float* stddev = (float*) volk_malloc(sizeof(float), alignment);
  *
- *   // Use a normal generator with 0 mean, stddev 1
+ *   // Use a normal generator with 0 mean, stddev 1000
```

Comment:
I know it is a small fix in our documentation. Still, I'd like to ask you to split such PRs. One for this doc update and one for the new generic kernel version. I'd say it is OK to keep this PR as is but it'd be great for future PRs. That would make reviews easier. e.g. I'd just merge the docs PR right away. Also, it makes the PR more concise.

Reply:
Sure, makes sense!